### ISSUE TITLE
Serve assets from CDN

### ISSUE THREAD

**Issue by katiemthom** [mbyczkowski/lacocina#157]

**Comment by katiemthom** [comment]
Looks like Greta and Fastly are provided as Heroku add-on options. 

https://elements.heroku.com/addons/greta
https://elements.heroku.com/addons/fastly

**Comment by katiemthom** [comment]
We'll make some other performance improvements before considering this.

**Comment by mbyczkowski** [comment]
I agree, let's see how things are going. I have also added appsignal add-on on lacocina-prod that should give us insights into the errors and performance (you should be able to access it with our heroku login).

Another option (definitely cheaper) would be to use S3 and CloudFront in front of it. Definitely a bit more work to get it going, but might be worth a shot.
